pilot/pkg/security/authz/builder/builder.go

```diff
@@ -151,6 +153,18 @@ type builtConfigs struct {
 	tcp  []*tcppb.Filter
 }
 
+func (b Builder) isDryRun(policy model.AuthorizationPolicy) bool {
+	dryRun := false
+	if val, ok := policy.Annotations[constants.DryRunAnnotation]; ok {
+		var err error
+		dryRun, err = strconv.ParseBool(val)
+		if err != nil {
+			b.option.Logger.AppendError(fmt.Errorf("failed to parse the value of %s: %v", constants.DryRunAnnotation, err))
```

Comment:
should we add this to validation webhook?

Reply:
yes, it's checked in pkg/config/validation/validation.go